PMSCore/Tester.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace PMSCore
+
+{
+    /// <summary>
+    /// A Tester user who managages test cases and bug reporting.
+    /// </summary>
+    public class Tester : UserBase
+    {
+        
+
+        // Properties specific to a Tester
+        public List<string> TestCases;
+        public List<(string Description, string Severity)> BugReports;
+
+        /// <summary>
+        /// Initializes a Tester and assigns username & TESTER permission level
+        /// </summary>
+        public Tester(string userName) : base(userName, Permission.TESTER)
+        {
+            TestCases = new List<string>();
+            BugReports = new List<(string, string)>();
+        }
+
+
+        public void SetPermission(Permission permission)
+        {
+            UserPermission = permission; 
+        }
+
+        public Permission GetPermission()
+        {
+            return UserPermission;
+        }
+
+        // Tester-Specific Methods
+
+        /// <summary>
+        /// Adds a new test case to the tester's list of test cases.
+        /// </summary>
+        public void AddTestCase(string testCaseDescription)
+        {
+            if (!string.IsNullOrWhiteSpace(testCaseDescription))
+            {
+                TestCases.Add(testCaseDescription);
+                Console.WriteLine($"Test case added: {testCaseDescription}");
```

Comment:
line 45:  Console.WriteLine($"Test case added: {testCaseDescription}"); 

- Console.WriteLine should be replaced with _logger.LogInformation for consistent logging